tests/sentry/api/endpoints/test_organization_sentry_app_installation_details.py

```diff
@@ -69,7 +69,7 @@ def test_no_access_outside_install_organization(self):
 class DeleteSentryAppInstallationDetailsTest(SentryAppInstallationDetailsTest):
     @responses.activate
     def test_delete_install(self):
-        responses.add(url="https://example.com/webhook", method=responses.POST, body={})
+        responses.add(url="https://example.com/webhook", method=responses.POST, body=b"")
```

Comment:
Raising the ignorable error before was masking a bug here where it required a bytes-like object